windows/src/engine/keyman32/Keyman32.cpp

```diff
@@ -653,6 +663,14 @@ BOOL ReleaseKeyboardMemory(LPKEYBOARD kbd)
 	return TRUE;
 }
 
+BOOL ReleaseKeyboardMemoryCore(km_kbp_keyboard* kbd)
+{
+  if (!kbd) return TRUE;
+  km_kbp_keyboard_dispose(kbd);
+  kbd = NULL;
```

Comment:
```suggestion
```

This is a no-op (could pass `**kbd` and then `*kbd = NULL` if you want to do this?)

Reply:
I don't understand? I want the memory pointed to by  `_td->lpKeyboards[i].coreKeyboard` to be deleted and then set the pointer to NULL

Reply:
> I don't understand? I want the memory pointed to by `_td->lpKeyboards[i].coreKeyboard` to be deleted and then set the pointer to NULL

The current code sets the pointer to NULL only in the current method - if you want the pointer to be NULL in the caller you'll have to pass in a pointer to the pointer.